### PR TITLE
Tweak Makefile

### PR DESCRIPTION
**Pull request by jkonecny12**
Strip dependencies from `Makefile` and move dependency resolving and testing environment setup to separate scripts.

### CODE REVIEW

**Comment by jkonecny12** [comment]
jenkins, test this please

**Comment by jkonecny12** [comment]
jenkins, test this please

**Comment by jkonecny12** [comment]
jenkins, test this please

**Comment by jkonecny12** [comment]
jenkins, test this please

**Comment by jkonecny12** [comment]
jenkins, test this please

**Comment by jkonecny12** [comment]
**UPDATED:**
* Small change in README
* Jenkins jobs are running correctly now.

File: scripts/testing/dependency_solver.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+            continue
+        result.add(pkg.strip())
+
+    return result
+
+
+def test_dependencies():
+    """Install all packages required for running the tests"""
+    result = set()
+    result.update(["e2fsprogs", "git", "bzip2", "cppcheck", "rpm-ostree", "pykickstart",
```
Comment (M4rtinK) [inline scripts/testing/dependency_solver.py]: Maybe this could be a constant ?

Comment (jkonecny12) [reply scripts/testing/dependency_solver.py]: Fixed.

**Pull request merged**